_posts/2017/2017-02-23-why-i-love-working-as-a-scrum-master.md

```diff
@@ -0,0 +1,136 @@
+---
```

Comment:
Can you briefly update me about the review status? AK, (Serena)?
It's still planned to publish tomorrow, right?